lib/temperature/index.js

```diff
@@ -22,7 +22,7 @@ try {
 var BMP085 = require('bmp085-sensor');
 } catch (err) {}
 
-var simulation = require('./simulator');
```

Comment:
it should be simulator that would be found in temperature/simulator.js

Reply:
well you complained about the fcat that there was a simulator already so I changed the name ...

Reply:
did I ? sorry I meant the opposite, the more code is aligned the best it is

Reply:
ok , Ill realling.